Test/integration-tests/cypress/e2e/examples/payment.cy.js

```diff
@@ -414,8 +414,8 @@ describe('Payments', function () {
             })
         })
     })
-
-    it('Klarna Pay by link', function () {
+    //skipping this test
+    it.skip('Klarna Pay by link', function () {
```

Comment:
Why are you skipping the tests?

Reply:
@emicha The tests have been fixed and reverted.